components/tokens/custom-express/custom-light-vars.css

```diff
@@ -14,5 +14,7 @@ governing permissions and limitations under the License.
 .spectrum--express.spectrum--light,
 .spectrum--express.spectrum--lightest {
   /* Drop Zone background color rgb */
-  --spectrum-drop-zone-background-color-rgb: var( --spectrum-indigo-800-rgb); /* var(--spectrum-accent-color-800);*/
+  --spectrum-drop-zone-background-color-rgb: var(--spectrum-indigo-800-rgb); /* var(--spectrum-accent-color-800);*/
+
+  --spectrum-steplist-current-marker-color-key-focus: var(--spectrum-blue-800);
```

Comment:
Since these values do not change between Spectrum and Express, we should be able to define them once in the `custom-spectrum` files, and not have to define them here again. So, this line can be removed.